src/zh/dev-guide/get_started/debug_tutorials.md

```diff
@@ -0,0 +1,68 @@
+# Goland+WSL2远程debug
+
+> - 已安装WSL2和较新版本的goland
+> - WSL2已安装并启动SSH服务
+
+## Windows侧操作
+
+1. clone项目
+
+2. **在goland配置SSH连接**
+   2.1 通过goland的菜单栏Tools–>Deployment–>Configuration来添加远程调试的配置
+   2.2 创建一个SFTP协议的配置，如图
+
+   <img title="" src="https://github.com/big-dust/openGemini.github.io/blob/main/static/img/dev-guide/get_started/debug_tutorials/deployment_connection.png?raw=true" alt="" style="zoom:67%;">
```

Comment:
use markdown image format instead of `<img>`

Reply:
ok.